pkg/license/download.go

```diff
@@ -60,7 +60,7 @@ func NewDownloaderWithOptions(opts *DownloaderOptions) (*Downloader, error) {
 type DownloaderOptions struct {
 	EnableCache       bool   // Should we use the cache or not
 	CacheDir          string // Directory where data will be cached, defaults to temporary dir
-	parallelDownloads int    // Number of license downloads we'll do at once
+	ParallelDownloads int    // Number of license downloads we'll do at once
```

Comment:
Do we need to make this public?

Reply:
Mmh I don't remember why I moved this public. Done.

Reply:
I don't recall why I made this public. I've reverted the change.